src/pages/version/TaskDuration.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from "react";
+import { useQuery } from "@apollo/client";
+import { pollInterval } from "constants/index";
+import { useToastContext } from "context/toast";
+import {
+  PatchTaskDurationsQuery,
+  PatchTaskDurationsQueryVariables,
+  PatchTasksQueryVariables,
+} from "gql/generated/types";
+import { GET_PATCH_TASK_DURATIONS } from "gql/queries";
+import { usePolling } from "hooks";
+import { useUpdateURLQueryParams } from "hooks/useUpdateURLQueryParams";
+import { TableControl } from "./TableControl";
+import { TaskDurationTable } from "./taskDuration/TaskDurationTable";
+
+interface Props {
+  taskCount: number;
+  queryVariables: PatchTasksQueryVariables;
```

Comment:
Why do we pass this in instead of calling the hook?

Reply:
Woops, possibly misunderstood your comment [here](https://github.com/evergreen-ci/spruce/pull/1211#discussion_r861210816). I put it back to the way it was